ups/drp_tasks.table

```diff
@@ -3,6 +3,7 @@
 # - Common third-party packages can be assumed to be recursively included by
 #   the "base" package.
 setupRequired(base)
+#setupRequired(gbdes)
```

Comment:
You clearly have more dependencies than are listed here: I count at least 6 in `wcsfit.py`.